index.js

```diff
@@ -0,0 +1,16 @@
+import { DateTime } from 'luxon/src/luxon.js';
```

Comment:
- [ ] The date is not being displayed because you have an error with regards to luxon, kindly check if you have correctly set the path, or if the luxon library is implemented 
![image](https://user-images.githubusercontent.com/62705947/228282631-0b21217d-c33e-4390-ae7e-896e698a9c48.png)

Changing to this did the trick for me `import { DateTime } from "./node_modules/luxon/src/luxon.js";`, but you will have some linter problems so I suggest to look for an alternative or fix the linters